config/src/cli.rs

```diff
@@ -0,0 +1,386 @@
+//! The relayer CLI and config definitions
```

Comment:
essentially all of the config methods are the same, just broken out into files now